app/controllers/api/v0/reading_collections_controller.rb

```diff
@@ -5,7 +5,6 @@ def create
         reading_collection = current_user.reading_collections.create(collection_params)
         articles = reading_collection.get_articles
         reading_collection.articles << articles
-        @reading_collection = reading_collection.to_json
```

Comment:
This was a group decision as we don't need to send anything back to FE from the endpoint.